### DISCUSSION TITLE
formatting comments inline comments?

### DISCUSSION THREAD

**Discussion by hellt** [yoheimuta/protolint#352] [Q&A] [answered]
Hi @yoheimuta 
thank you for the protolint
I know that protolint is not a formatted, but a linter, but still worth asking, do you have plans to format proto documents with regards to inline comments, which are often seen in protos.

```diff
message CommitConfirmedRequest {
  CommitConfirmedAction action = 1;
-  uint32 timeout = 2; // timeout for START, or new timeout value
-  uint64 commit_id = 3; // CommitId from commit for ACCEPT or REJECT
+  uint32 timeout = 2;    // timeout for START, or new timeout value
+  uint64 commit_id = 3;  // CommitId from commit for ACCEPT or REJECT
}
```

**Comment by yoheimuta** [comment, marked as answer]
@hellt Sorry for the late reply. I hadn't noticed this discussion.
I have no plans, but formatting inline comments sounds useful and the PR is welcome.
